### ISSUE TITLE
Vertical divider

### ISSUE THREAD

**Issue by NekoJonez** [WordPress/gutenberg#51286]
## What problem does this address?
Sometimes, I like to work with columns to place text on a page. Yet, I sometimes want to place a line between the two columns. A vertical line. I could do that with a table and giving the outer borders the same color as the background... Yet, that's silly imho. So...

## What is your proposed solution?
I think it would be cool if in the options of the divider you have an option to flip it vertical. And that this only works in columns or tables. And the divider is then as long as the columns. 
Or if this doesn't work as an option, maybe a new block?

**Comment by bph** [comment]
The core columns blog has border support, too. You could accomplish a separation by using a left and right side border on the middle column if you are dealing with a three column layout. Using this method, also gives you control over the padding and margin 
<img width="1223" alt="Screenshot 2023-06-07 at 11 47 20" src="https://github.com/WordPress/gutenberg/assets/39980/fb8e43c7-c3d7-4282-b2f6-1ed647918f85">

Does that look like a feasible way to move forward?

**Comment by bph** [comment]
Here is the front end: (would need some slight adjustments on the padding and margins) 
<img width="1200" alt="Screenshot 2023-06-07 at 11 50 13" src="https://github.com/WordPress/gutenberg/assets/39980/f5e644df-fc17-4b8c-85d6-50cf0c174762">

**Comment by NekoJonez** [comment]
I'm either overlooking something OR this is something that didn't get to core yet... 
Since when I click in WordPress 6.2.2 with the Neve theme... I don't see the options you show. 

![image](https://github.com/WordPress/gutenberg/assets/4569581/10a615f1-b424-4530-9d30-392848c07089)

Or is this because I'm only using two columns?

**Comment by NekoJonez** [comment]
@bph I just went testing via LocalWP and it looks like what you are showing isn't in WordPress v6.2.2... But in v6.3. 
I ran the beta testing plugin and without installing the Gutenberg plugin, I show these options in the editor. 

And yeah, for my use-case these options will help. :) 

So, I'll just have to wait for the full release of WP v6.3 to have this feature to play with.

Should we leave this issue open for people if they have other use cases OR close it?

**Comment by NekoJonez** [comment]
Hrm, it seems that these options don't appear on non FSE themes...

**Comment by NekoJonez** [comment]
@bph , sorry for the earlier ping... I had to delete the comment since it was untrue from later testing.

So, the features you have shown DO appear but only in site-editor/FSE themes. If you use a non FSE theme like Neve, Astra, SpicePress, Zakra... These options don't show up. 
Is that a bug in Gutenberg or a limitation of the themes?

**Comment by NekoJonez** [comment]
When Twenty Twenty-Three is active:
![image](https://github.com/WordPress/gutenberg/assets/4569581/cd05a2e7-32f2-4970-8388-4b2e04e68d8f)

When Neve theme is active:
![image](https://github.com/WordPress/gutenberg/assets/4569581/765883ac-c396-4c6d-87ed-53fb1c0ce4bf)

**Comment by bph** [comment]
Non-block themes need to enable the _appearancetools_  via their functions.php

``` add_theme_support( 'appearance-tools' );```

It's not.made available out of the box as there are a few edge cases with classic themes. 
Documentation: https://developer.wordpress.org/block-editor/how-to-guides/themes/theme-support/#appearance-tools

You could test via a child theme if your site can work with those added tooling.

**Comment by NekoJonez** [comment]
I tried to add this in a child theme, but I'm not well versed enough in PHP to fix this.
Curious, why are these features for block themes only? Since otherwise it might give the impression that we push people towards the block themes...
